transformations/word_noise/transformation.py

```diff
@@ -0,0 +1,101 @@
+from random import Random
```

Comment:
I think the test cases in main are failing because of this. Can you use np.random and create a PR to fix that please?

Reply:
Sure thing.

Reply:
I created a temporary fix here: https://github.com/GEM-benchmark/NL-Augmenter/pull/195/files

Reply:
Thank you!